tvb_storage/tvb/tests/storage_test.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
```

Comment:
I suggest to put another layer `tvb.tests.storage`, then storage_test.py and all the others

Reply:
But first, the tvb_storage folder structure will be changed to: **tvb/storage** that contains a folder called **h5** with all this code inside and a folder **tests** with the structure suggested by Lia. Near these, I would see the new file with the tvb-storage interface/API as we discussed.